inference-engine/ie_bridges/java/samples/benchmark_app/Main.java

```diff
@@ -94,22 +145,27 @@ public static void main(String[] args) {
             System.exit(1);
         }
 
+        int step = 0;
+        final int nsteps = 11;
+
         // ----------------- 1. Parsing and validating input arguments ---------------------------------------------
-        System.err.println("Parsing and validating input arguments");
+        System.err.println("[Step " + ++step + "/" + nsteps + "]" + " Parsing and validating input arguments");
 
         ArgumentParser parser = new ArgumentParser(args);
 
         String xmlPath = parser.get("-m", null);
         String device = parser.get("-d", "CPU");
-        int nireq = parser.getInteger("-nireq", 1);
-        int niter = parser.getInteger("-niter", 1000);
-        int batchSize = parser.getInteger("-b", 1);
-        int nthreads = parser.getInteger("-nthreads", -1);
+        int nireq = parser.getInteger("-nireq", 0);
+        int niter = parser.getInteger("-niter", 0);
+        int batchSize = parser.getInteger("-b", 0);
+        int nthreads = parser.getInteger("-nthreads", 0);
+        int nstreams = parser.getInteger("-nstreams", 0);
+        int timeLimit = parser.getInteger("-t",0);
```

Comment:
Why zeros?

Reply:
Like in c++ version. Because later we need to check if this values were set in command line and set optimal values.

Reply:
So all these values are zeroed by default in C++ as well?

Reply:
![image](https://user-images.githubusercontent.com/32199725/87771875-9f4c1800-c829-11ea-8253-ec903c6ee023.png)
Yes, only ```nstreams``` is an empty string. See ```inference-engine/samples/benchmark_app/benchmark_app.hpp```